src/event/handlers/CommandParserHandler.ts

```diff
@@ -15,6 +16,12 @@ class CommandParserHandler extends EventHandler {
 
 	handle = async (message: Message): Promise<void> => {
 		if (message.content.startsWith(COMMAND_PREFIX)) {
+			const BOTLESS_CHANNELS = getConfigValue("BOTLESS_CHANNELS");
```

Comment:
I wonder if this has a worse performance than a regular import, but my Node knowledge is still incipient, please correct me if I'm wrong. I thought it would be better to move this line up along with the imports, but apparently `getConfigValue()` would be only called once during the program runtime and then the stub wouldn't cause any effect.

Reply:
I'm not too worried about performance right now, honestly. Should be fine as is.